bitnami/redis/values.yaml

```diff
@@ -50,6 +50,12 @@ extraDeploy: []
 ## @param useHostnames Use hostnames internally when announcing replication
 ###
 useHostnames: true
+## @param useHostnamesThreshold Failure threshold for internal hostnames collection
+###
+useHostnamesThreshold: 5
+## @param useHostnamesTimeout Timeout seconds between probes for internal hostnames collection
+###
+useHostnamesTimeout: 5
```

Comment:
As an user I wouldn't understand very well the use of these variables. If `useHostnames` is false then we will retrieve the IP addresses for the mongodb instances and only in that case `useHostnamesThreshold` and `useHostnamesTimeout` makes sense but `useHostnames` has to be false... I have to say that I am very bad naming variables and functions but I think something like this will be clearer for the final user:
```suggestion
## @param useHostnames Use hostnames internally when announcing replication. If false, the hostname will be resolved to an IP address.
###
useHostnames: true
## @param nameResolutionThreshold Failure threshold for internal hostnames resolution.
###
nameResolutionThreshold: 5
## @param nameResolutionTimeout Timeout seconds between probes for internal hostnames collection.
###
nameResolutionTimeout: 5
```
What do you think?

Reply:
Yes, I'm bad at naming variables too :)
Your suggestion sounds much better